app/assets/javascripts/discourse-common/addon/lib/icon-library.js

```diff
@@ -51,6 +51,9 @@ export const REPLACEMENTS = {
   "notification.reaction": "bell",
   "notification.votes_released": "plus",
   "notification.chat_quoted": "quote-right",
+  "user_menu.replies": "reply",
+  "user_menu.drafts": "pencil-alt",
+  "sidebar.all_categories": "list",
```

Comment:
btw, similar to /t/124272 would be nice if the icon names were consistent in terms of `-` vs `_` 😅 
but that's for another PR 😉

Reply:
The reason I went with underscore here is because thats how we do most yaml settings. Though this isnt "yaml" per se, it resembles the way we do other things in Discourse.